builder/builder.go

```diff
@@ -78,7 +108,7 @@ func (b *Builder) Build() error {
 	carto := mss.New()
 
 	for _, mss := range b.mss {
-		err := carto.ParseFile(mss)
```

Comment:
ParseFile should be used if the file is known to get errors with correct line numbers.